Modules.md

```diff
@@ -108,8 +108,9 @@ to allow *explicitly* sharing linear memory between multiple modules.
 ## Initial state of linear memory
 
 A module will contain a section declaring the linear memory size (initial and
-maximum size allowed by `sbrk`) and the initial contents of memory (analogous
-to `.data`, `.rodata`, `.bss` sections in native executables).
+maximum size allowed by [`resize_memory`](AstSemantics.md#resizing) and the
+initial contents of memory (analogous to `.data`, `.rodata`, `.bss` sections in
+native executables).
```

Comment:
Do we want to dictate these, or let wasm handle them but restrict where they can go? As discussed in #285 we can get a no-signal non-page-table but highly-efficient zero-page and `.rodata` implementation.